src/test/resources/edu/cornell/kfs/concur/rest/xmlObjects/fixture/travel-request-detail-dto.xml

```diff
@@ -0,0 +1,93 @@
+<TravelRequestDetails xmlns="http://www.concursolutions.com/api/travelrequest/2012/06" xmlns:i="http://www.w3.org/2001/XMLSchema-instance">
```

Comment:
This file contains several "key" fields ("CashAdvanceKey", "RequestKey", etc.). Are any of those fields considered sensitive.

Reply:
I don't think so, it is just a hashed value, but it might be a good idea to clear it out

Reply:
Thanks, the test file looks good now.